folioreader/src/main/java/com/folioreader/util/FolioReader.java

```diff
@@ -80,4 +105,12 @@ private Intent getIntentFromUrl(String assetOrSdcardPath, int rawId, Context con
         }
         return intent;
     }
+
+    public void setOnHighlightCreateListener(OnHighlightCreateListener onHighlightCreateListener) {
+        this.onHighlightCreateListener = onHighlightCreateListener;
+    }
+
+    public void unSubscribe() {
```

Comment:
- rename with unregisterHighlightListener
- also nullify onHighlightListener

`this.onHighlightCreateListener = null;`